src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -543,7 +543,7 @@ def call(
         # So we can broadcast to [batch_size, num_heads, from_seq_length, to_seq_length]
         # this attention mask is more simple than the triangular masking of causal attention
         # used in OpenAI GPT, we just need to prepare the broadcast dimension here.
-        extended_attention_mask = inputs["attention_mask"][:, tf.newaxis, tf.newaxis, :]
+        extended_attention_mask = tf.expand_dims(tf.expand_dims(inputs["attention_mask"], axis=1), axis=1)
```

Comment:
Could we do a `tf.reshape` here? Two `expand_dims` in a row is a bit hard to read (and the result is not super understandable).